src/lib.rs

```diff
@@ -72,13 +72,44 @@ macro_rules! kernel_module {
         )*
     };
 
-    (@attribute $name:ident, $value:expr) => {
+    /* TODO: The modinfo attributes below depend on the compiler placing
```

Comment:
Rust generally uses `//` for comments. i'm surprised this even worked.

Reply:
You can tell I've been writing lots of C at work.